examples/game/lib/layer.dart

```diff
@@ -0,0 +1,19 @@
+part of sprites;
+
+class Layer extends Node with SpritePaint {
+  Paint _cachedPaint = new Paint()
+    ..setFilterQuality(FilterQuality.low)
+    ..isAntiAlias = false;
+
+  void _prePaint(PaintingCanvas canvas, Matrix4 matrix) {
+    super._prePaint(canvas, matrix);
+
+    _updatePaint(_cachedPaint);
+    canvas.saveLayer(null, _cachedPaint);
```

Comment:
This is my first pass, so still working on the details.

I'm thinking it could be useful for devs to be able to pass in a custom rect, automatically computing the bounds would probably be pretty tricky.